apps/web/app/(app)/simple/SimpleList.tsx

```diff
@@ -78,12 +93,15 @@ export function SimpleList(props: {
                 userEmail={props.userEmail}
                 toHandleLater={toHandleLater}
                 onSetToHandleLater={onSetToHandleLater}
+                handleUnsubscribe={() => handleUnsubscribe(message.id)}
               />
             );
           })}
-        {props.messages.length === 0 && (
-          <Celebration message="All emails handled!" />
-        )}
+        {props.messages.filter(
+          (m) =>
+            !toHandleLater[m.id] &&
+            !unsubscribed.find((unsub) => unsub.id === m.id),
+        ).length === 0 && <Celebration message="All emails handled!" />}
```

Comment:
Looks like this is a copy paste of the above. Can do `const filteredMessages = ...` instead and then use this in each spot.